content/software-security/cves/infamous-cves.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Infamous Software Vulnerabilities"
+description: "An overview a few of the most critical, widespread, and impactful known software vulnerabilities"
+lead: "An overview a few of the most critical, widespread, and impactful known software vulnerabilities"
+type: "article"
+date: 2023-07-21T19:16:39+00:00
+lastmod: 2023-07-25T17:17:48+00:00
+contributors: ["Michelle McAveety"]
+draft: false
+tags: ["CVE", "Overview"]
+images: []
+menu:
+  docs:
+    parent: "cves"
+weight: 003
+toc: true
+---
+
+[Software vulnerabilities](/software-security/cves/cve-intro/) vary in their severity – some are difficult to exploit and have minimal implications, while others can be exploited easily and give an attacker significant leverage over a computer system. In cases where widely-implemented software contains high-severity vulnerabilities, the damage caused by their exploitation can affect millions of developers and services worldwide.
+
+In this article, you will learn how the KEV Catalog tracks known exploited software vulnerabilities, and how it serves as a tool for developers and federal agencies. In addition, you will explore Log4Shell, Heartbleed, and Shellshock, three infamous software vulnerabilities which have had major impacts on software security worldwide.
+
+
+## CISA KEV Catalog
+
+The U.S. Cybersecurity and Infrastructure Security Agency (CISA) operates the Known Exploited Vulnerabilities (KEV) Catalog, which is populated with CVEs that have existing exploits “in the wild”. The KEV Catalog serves as a tool for developers as it identifies CVEs that need to be prioritized for remediation because of their exploitability status. Federal civilian executive branch agencies [must remediate vulnerabilities](https://www.cisa.gov/news-events/directives/binding-operational-directive-22-01) present in the KEV Catalog by a due date specified by the CISA. Focusing on patching out these vulnerabilities limits the ability of attackers to find a potential known route into a system.
```

Comment:
Maybe we can add agencies and things like the KEV Catalog to the Chainguard Glossary?